app/__init__.py

```diff
@@ -0,0 +1,12 @@
+# app/__init__.py
+
+from flask import Flask
+
+# Initialize the app
+app = Flask(__name__, instance_relative_config=True)
+
+# Load the views
+from app import views
```

Comment:
views.py is imported after app because it uses the Flask object inside 'from app import views`. There is an import error if views is imported before app is created.